### PR TITLE
Add a reference to Braille Computer Notation of the UKAAF

### PR DESCRIPTION
**Pull request by egli**

### CODE REVIEW

File: english/README.md [hunk 1]
```diff
@@ -34,3 +34,11 @@
 [BRF version](http://brailleauthority.org/mathscience/nemeth1972.zip)
 
 [Source](http://brailleauthority.org/mathscience/math-science.html)
+
+## [Braille Computer Notation](https://www.ukaaf.org/wp-content/uploads/2020/03/Braille-Computer-Notation-PDF.pdf)
+
+- _by_: Braille Authority of the United Kingdom
+- _published_: 1996, 2006
+- _language_: English
```
Comment (bertfrees) [inline english/README.md]: I'm going to change this to:

> _by_: Braille Authority of the United Kingdom
> _applies to_: United Kingdom
> _published_: 1996, 2006, printed by Royal National Institute for the Blind, Peterborough
> _language_: English

File: english/README.md [hunk 2]
```diff
@@ -34,3 +34,11 @@
 [BRF version](http://brailleauthority.org/mathscience/nemeth1972.zip)
 
 [Source](http://brailleauthority.org/mathscience/math-science.html)
+
+## [Braille Computer Notation](https://www.ukaaf.org/wp-content/uploads/2020/03/Braille-Computer-Notation-PDF.pdf)
```
Comment (bertfrees) [inline english/README.md]: Should not be under the section "US English".

Comment (egli) [reply english/README.md]: What do you mean? It is on the same level as _Unified English Braille_ and _US English: BANA_. Are you saying it should not be third after UEB and BANA?

File: english/README.md (hunk 1 above)
Comment (egli) [reply english/README.md]: I just did it

File: english/README.md (hunk 2 above)
Comment (bertfrees) [reply english/README.md]: Ah yes I see. But I would put it under a section "UK English: BAUK" or something.

Comment (egli) [reply english/README.md]: Done

**Pull request merged**